app/javascript/src/Common/SearchableSelect.jsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+import { TextInput } from '@patternfly/react-core'
```

Comment:
I'm not sure if we should include patternfly/react stuff here. 
At the moment we are waiting for a definition about IE11 support, to define if using patternfly 4 or not.
We should keep this PR without any PF4 dependency, IMO
What do you think @thomasmaas ?

Reply:
Well, we will have to use PF4. If this works and is tested in IE11 I don't see a problem.

Reply:
I think I will remove PF4 from this, it doesn't make that much sense. Specially when were are faking it in some other places.